fastapi/applications.py

```diff
@@ -36,6 +36,9 @@ def __init__(
         title: str = "FastAPI",
         description: str = "",
         version: str = "0.1.0",
+        terms_of_service: Optional[str] = "",
+        contact: Optional[Dict[str, Union[str, Any]]] = None,
```

Comment:
Don't we need to validate the `keys` on the `contact` and `license_info`?

Reply:
We could, but that is not really done for other arguments either.